GodotModding/godot-mod-loader
# Only Interface PR

split the pr #33 into two parts for only self installation and only UI.
the loader now falls back to default behaviour if the `interface` folder is not present. that allows the user to just deselect that folder when installing from the asset lib

to test:
1. Redoing the loader setup via button and restarting
2. Toggling mods on and off (use these two small ones that only print stuff for you to test with: [download mods.zip](https://github.com/GodotModding/godot-mod-loader/files/10421417/mods.zip)). A disabled mod should not be initialised upon game start.
3. skipping the selection screen (via flag, pressing <kbd>ALT</kbd> or via the option in the selection)

Stuff that's not done yet:
1. mod icons
2. flag for autoload mods that are always loaded right away (state can only be changed when the game is restarted)
